package.json

```diff
@@ -12,16 +12,16 @@
     "test": "jest",
     "snyk": "snyk test",
     "lint": "eslint -c .eslintrc.js --ignore-path .eslintignore",
-    "lint.test": "eslint -c .test.eslintrc.js  --ignore-path .test.eslintignore",
-    "coverage": "yarn test --runInBand --ci --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "lint.test":
```

Comment:
We have to fix this formatting conflict soon, has more changes unrelated to the PR than related :/.

Reply:
Definitely! I created #698 so we handle that.